### DISCUSSION TITLE
No puedo loguear en https://web.staging.cotalker.com

### DISCUSSION THREAD

**Discussion by seiturra** [Cotalker/documentation#201] [4. Bug Reports] [answered]
Buenas,

Estoy trabajando en un proyecto en el ambiente staging y no puedo hacer login. Parece haber algún servicio caído porque web client sí responde pero al intentar iniciar con las credenciales, se indica ¡Usuario y/o contraseña incorrectos! y en la llamada de red responde con un 503 Service Temporarily Unavailable. 

Saludos.

**Comment by guillevalin** [comment, marked as answer]
@SeIturra un nodo del cluster está teniendo problemas de conectividad.

Se solucionará el problema en aproximadamente 10 minutos.

**Reply by seiturra** [reply]
Buena! Gracias @guillevalin
